openslides_backend/action/action.py

```diff
@@ -181,16 +185,39 @@ def handle_relation_updates(
         )
         fields: Optional[Dict[str, Any]]
         for fqfield, data in relation_updates.items():
+            field = getattr(model_registry[fqfield.collection], fqfield.field, None)
```

Comment:
You can use `self.model.get_field` to access the field.

Reply:
It's not necessary the self.model, sometimes the collection from the other side of the relation

Reply:
doesn't exist anymore